.github/workflows/codeql.yml

```diff
@@ -0,0 +1,66 @@
+name: 'CodeQL'
+
+on:
+  push:
```

Comment:
Shouldnt this run on all branches pull and push?

Reply:
I mean can we dynamically just include all future branches?

Reply:
I think so, updating it to all branches.